### DISCUSSION TITLE
How to format the Hoogle output

### DISCUSSION THREAD

**Discussion by QRCoding** [mrcjkb/haskell-tools.nvim#308] [Q&A] [answered]
Is there any way to have the Hoogle output formatted (thereby removing the markup tags)? See picture.

![hoogle](https://github.com/mrcjkb/haskell-tools.nvim/assets/12382311/665cff7e-bdd1-490b-a42d-0e3aa07ae040)

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave: 

I have recently toyed with the idea of contributing markdown support to hoogle (see #296). This would give the previews tree-sitter syntax highlighting.
Hoogle uses blaze though, so I would have to write a blaze-markdown library first.

A quick and dirty solution would be to check if pandoc is installed, and if so, pipe the preview text through pandoc (converting it from HTML to markdown).

But something like that will have to be implemented in haskell-tools.
